changes to startup and sample app doc as part of review when attempti…

gabemontero opened the pull request:
…ng to repro issue 2740

@bparees 
Ben - here is my take on the doc updates we discussed via email.  In case this context helps, I went back and forth on some form of update that warns the user about HW/OS enablement for virtualization.  Hopefully you agree that the RHEL link I provided is in line with the flavor of hint you were comfortable with.

bparees commented on CONTRIBUTING.adoc:
```diff
@@ -15,10 +15,10 @@ Or if you are interested in development, start with:
 * <<openshift-development>>
 
-And choose between:
+	And choose between:
```
this indent is making markdown interpret it as a code/preformatted blog.  i'm not a markdown guru but you'll need to find another way to do this.

bparees commented on examples/sample-app/README.md:
```diff
@@ -51,6 +51,9 @@ This will set up a go workspace locally and will build all go components.  It is
 ### Docker Changes ###
 
+**VAGRANT USERS**:
+If you are using a Vagrant image you can skip this step.
```
s/a/the/ since it's not true for just any vagrant image, only ours.

bparees commented on examples/sample-app/README.md:
```diff
@@ -68,6 +71,9 @@ These instructions assume you have not changed the kubernetes/openshift service
 ### FirewallD Changes ###
 
+**VAGRANT USERS**:
+If you are using a Vagrant image you can skip this step.
```
s/a/the/ since it's not true for just any vagrant image, only ours.

bparees commented on examples/sample-app/README.md:
```diff
@@ -163,26 +169,17 @@ This section covers how to perform all the steps of building, deploying, and upd
-
-
-7. Bind a user names `test-admin` to the `view` role in the default namespace so you can observe progress in the web console
+6. Bind a user names `test-admin` to the `view` role in the default namespace so you can observe progress in the web console
```
I think this is unnecessary now too.  later in the steps they create a non-default project and create their app there, so there's no need to grant them access to the default project.

Note that until that project is created, they won't be able to see anything in the web console if they login as test-admin.

bparees commented on examples/sample-app/README.md:
```diff
@@ -163,26 +169,17 @@ This section covers how to perform all the steps of building, deploying, and upd
 
 
-9. *Optional:* View the OpenShift web console in your browser by browsing to `https://<host>:8443/console`.  Login using the user `test-admin` and any password.
+8. *Optional:* View the OpenShift web console in your browser by browsing to `https://<host>:8443/console`.  Login using the user `test-admin` and any password.
```
Delete this step.  somehow we ended up with two of them?   (this one and step 10).  Anyway it makes more sense to have them login to the console after creating the project in step 9. that way they will actually be able to see something when they login.

bparees commented:
@gabemontero the link looks fine, couple other comments on the steps though.

gabemontero commented:
OK, I've incorporated the responses, as well as went back and added the new oc command for processing the json file that Cesar pointed us to yesterday.

bparees commented:
@gabemontero have you walked through this finalized set of steps to confirm it all works as written?

gabemontero commented:
I've done the non-optional ones.  I'll get started on the optional/advanced ones, but tell me if that is overkill.

bparees commented:
yeah that's probably overkill.

@smarterclayton @danmcp this is a docs only change and it's a pretty useful one, can we get it approved please?

smarterclayton commented:
Approved

bparees commented:
[merge]

gabemontero commented:
@bparees OK Ben, I've pushed the changes from the Advanced section walkthrough.  The "fixes stemming from advanced section walkthrough"

bparees commented:
one minor issue and needs a squash

bparees commented:
@gabemontero you've got a merge conflict now (evidently someone else changed this file while you were working on it).  i'll show you how to deal w/ that tomorrow.

gabemontero commented:
OK thanks.  Yep, also got a message from the Openshift bot saying a rebase had to occur in order to merge.

----- Original Message -----
From: "Ben Parees" <notifications@github.com>
To: "openshift/origin" <origin@noreply.github.com>
Cc: "Gabe Montero" <gmontero@redhat.com>
Sent: Tuesday, June 9, 2015 6:03:59 PM
Subject: Re: [origin] changes to startup and sample app doc as part of review when attempti… (#2969)

@gabemontero you've got a merge conflict now (evidently someone else changed this file while you were working on it).  i'll show you how to deal w/ that tomorrow.


---
Reply to this email directly or view it on GitHub:
https://github.com/openshift/origin/pull/2969#issuecomment-110518053

bparees commented:
[merge]
